lib/instrumental/agent.rb

```diff
@@ -77,10 +78,15 @@ def initialize(api_key, options = {})
       @certs           = certificates
       @dns_resolutions = 0
       @last_connect_at = 0
+      @metrician       = options[:metrician].nil? ? true : !!options[:metrician]
```

Comment:
this seems more like it should be:

`options.has_key?(:metrician) ? !!options[:metrician] : true`, which really boils down to `!!options[:metrician]` as does what you've written originally.